app/db.py

```
@@ -20,43 +21,39 @@ def parse_jdbc_mysql_url(jdbc_url: str):
+    host = os.getenv("DB_HOST")
+    port = os.getenv("DB_PORT")
     user = os.getenv("DB_USERNAME")
     pwd  = os.getenv("DB_PASSWORD")
-    if not (jdbc_url and user and pwd):
-        raise RuntimeError("DB_URL/DB_USERNAME/DB_PASSWORD 환경변수를 설정하세요.")
+    db   = os.getenv("DB_NAME")
 
-    host, port, db, params = parse_jdbc_mysql_url(jdbc_url)
+    # 모든 필수 환경 변수가 있는지 확인
+    if not all([host, port, user, pwd, db]):
+        raise RuntimeError("DB_HOST/DB_PORT/DB_USERNAME/DB_PASSWORD/DB_NAME 환경변수를 설정하세요.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)